testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -1997,123 +1993,6 @@ public void testPaginationDisallowedLimitedSupport() throws IOException {
             "{\"description\":\"The results as requested must fit in one page, try increasing the `page-size` parameter.\",\"code\":400}");
   }
 
-  // Below are "namespace" tests that should match the REST Api's keyspace tests
-  @Test
-  public void getNamespaces() throws IOException {
```

Comment:
Could we delay this change until #937 is merged, please? :)

Reply:
Sure, waiting :clock1:

Reply:
@dimas-b need your approval to merge :pray:

Reply:
sorry, forgot about this yesterday.... approving